crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -501,12 +505,26 @@ pub struct AuthorityEpochTables {
 
     /// Records messages processed from other nodes. Updated when receiving a new dkg::Message
     /// via consensus.
-    pub(crate) dkg_processed_messages: DBMap<PartyId, dkg::ProcessedMessage<PkG, EncG>>,
+    pub(crate) dkg_versioned_processed_messages: DBMap<PartyId, VersionedProcessedMessage>,
```

Comment:
following the pattern for other fields where we've added versionining to messages that are part of these tables, these should end in a _v2 suffix rather than having "versioned" in the name

(I'm not sure I actually like that better but it seems best to stick with current conveniton rather than adding a second one?)